include/rtc.h

```diff
@@ -137,6 +141,25 @@ static inline int rtc_get_pending_int(struct device *dev)
 	return api->get_pending_int(dev);
 }
 
+/**
+ * @brief Function to get RTC ticks per second
+ *
+ * The function is to return the RTC ticks per seconds.
+ * The caller can use the return value to convert RTC ticks to the desired time
+ * spec.
+ *
+ * @param dev Pointer to the device structure for the driver instance.
+ *
+ * @retval How many RTC ticks in one second.
+ */
+static inline int rtc_get_ticks_per_sec(struct device *dev)
```

Comment:
If we add this function I think we need to deprecate the following macro
```
 #define RTC_ALARM_SECOND (32768 / (1UL << RTC_DIVIDER))
```
since its enforcing ticks per second value to 32768.

Reply:
Agree. this one should be deprecated.